rules/cross-platform/threat_intel_module_match.toml

```diff
@@ -0,0 +1,144 @@
+[metadata]
+creation_date = "2021/04/21"
+maturity = "production"
+updated_date = "2021/04/21"
+
+[rule]
+author = ["Elastic"]
+description = """
+This rule is triggered when indicators from the Threat Intel Filebeat module has a match against local file or network observations.
+"""
+from = "now-10m"
+index = ["auditbeat-*", "endgame-*", "filebeat-*", "logs-*", "packetbeat-*", "winlogbeat-*"]
+interval = "9m"
+language = "kuery"
+license = "Elastic License v2"
+name = "Threat Intel Filebeat Module Indicator Match"
+note = """ ## Triage and Analysis
+If an indicator matches a local observation, the following enriched fields will be generated to identify the indicator, field, and type matched.
+
+- `threatintel.indicator.matched.atomic` - this identifies the atomic indicator that matched the local observation
+- `threatintel.indicator.matched.field` - this identifies the indicator field that matched the local observation
+- `threatintel.indicator.matched.type` - this identifies the indicator type that matched the local observation
+"""
+references = [ "https://www.elastic.co/guide/en/beats/filebeat/current/filebeat-module-threatintel.html"]
+risk_score = 99
+rule_id = "dc672cb7-d5df-4d1f-a6d7-0841b1caafb9"
+severity = "critical"
+tags = ["Elastic", "Windows", "Elastic Endgame", "Network", "Continuous Monitoring", "SecOps", "Monitoring"]
+type = "threat_match"
+
+threat_index = [ "filebeat-*"]
```

Comment:
it's kinda too bad that this all get flattened, but that's more of a kibana API thing
i think it would be nicer if the fields were all namespaced by type instead of jammed together